_posts/2025-03-27-infinispan-15.2.0.Final.adoc

```diff
@@ -0,0 +1,55 @@
+---
+layout: blog
+title: Infinispan 15.2.0.Final
+permalink: /blog/:year/:month/:day/infinispan-15-2
+date: '2025-03-27T00:00:00.000-00:00'
+author: ttarrant
+tags: [ "release", "final" ]
+---
+
+= Infinispan 15.2.0.Final
+
+*_"Feelin' Blue"_*
+
+It's new.
+It's blue.
+It's Infinispan 15.2, codenamed https://untappd.com/b/three-sisters-brewery-feelin-blue/6087587["Feelin' Blue"].
+
+[caption="Feelin' Blue"]
+image::/assets/images/blog/feelinblue.png[Feelin' Blue, 415, 975, align="center"]
+
+Without further ado, here's a list of what's new (and I promise to stop with the Dr. Seuss rhyming):
+
+== RESP endpoint
+=== JSON
+
+Infinispan now implements the https://redis.io/docs/latest/develop/data-types/json/[Redis JSON API].
+
+=== Lua scripting
+
+It is now possible to run https://redis.io/docs/latest/develop/interact/programmability/eval-intro/[Lua scripts].
+The big advantage is that this works in Infinispan clusters, without having to worry about key ownership.
+
+== Fixes
+Lots of these. However, should you find something that doesn't work, head over to our https://github.com/infinispan/infinispan/issues[issues] and provide any details that help us debug and solve problems.
+
+== JDK requirements
+Like the previous 15.x releases, you will need at least JDK 17 in order to use Infinispan 15.2. Infinispan also supports the recently released JDK 24.
+
+== Deprecations and removals
```

Comment:
could we mention something about the Console new look ? and add a screenshot ? I need to update the website too

Reply:
I added two screenshots